src/components/content-header-card/index.tsx

```diff
@@ -0,0 +1,184 @@
+	const hasLinks = links && links.length > 0
+	const hasButtons = buttons && buttons.length > 0
+	return (
+		<Card
+			elevation="base"
+			className={classNames(s.contentHeaderCard, className)}
+		>
+			<div className={s.cardTop}>
+				{icon && (
+					<IconTileLogo className={s.icon} size="large" productSlug={icon} />
+				)}
+				<div className={s.content}>
+					<div className={s.upper}>
+						<div className={s.left}>
+							<div className={s.titleWrap}>
+								<Heading size={300} weight="bold" level={1} className={s.title}>
+									{title}
+								</Heading>
+								{attribution && (
+									<Text size={100} weight="medium" className={s.attribution}>
+										{attribution}
+									</Text>
+								)}
+							</div>
+							{description && (
+								<Text size={200} className={s.description}>
+									{description}
+								</Text>
+							)}
+						</div>
+						{dropdown && (
+							<DropdownDisclosure
+								listPosition="right"
+								className={s.dropdown}
+								color="secondary"
+								text={dropdown.text}
+							>
+								{dropdown.items.map((item: DropdownItem) => {
+									return (
+										<DropdownDisclosureLinkItem
+											key={item.text}
+											href={item.href}
+										>
+											{item.text}
+										</DropdownDisclosureLinkItem>
+									)
+								})}
+							</DropdownDisclosure>
+						)}
+					</div>
+					{(note || badges) && (
+						<div className={s.lower}>
+							{badges && (
+								<BadgeList
+									className={s.badgeList}
+									type="outlined"
+									size="small"
+									badges={badges}
+								/>
+							)}
+							{note && (
+								<Text size={200} className={s.note}>
+									{note}
+								</Text>
+							)}
+						</div>
+					)}
+				</div>
+			</div>
+			{(hasLinks || hasButtons) && (
+				<div className={s.cardBottom}>
+					{hasLinks && (
+						<ul className={s.linksList}>
+							{links.map((link: Link) => {
+								return (
+									<li key={link.text}>
+										<StandaloneLink
+											text={link.text}
+											icon={link.icon}
+											href={link.href}
+											opensInNewTab={!link.href.startsWith('/')}
+											iconPosition="leading"
+										/>
+									</li>
+								)
+							})}
+						</ul>
+					)}
+					{hasButtons && (
+						<ul
+							className={classNames(s.buttonsList, {
+								[s.buttonsOnly]: !hasLinks,
+							})}
+						>
+							{buttons.map((button: Button, index: number) => {
+								return (
+									<li key={button.text}>
+										<Button
+											onClick={(e) => {
+												e.preventDefault()
+												button.onClick()
+											}}
+											text={button.text}
+											color={
+												button.isPrimary && index === 0
+													? 'primary'
+													: 'secondary'
+											}
+											icon={button.icon}
+											iconPosition="leading"
+											size="small"
+										/>
+									</li>
+								)
+							})}
+						</ul>
+					)}
```

Comment:
🙋🏻 Should this support `ButtonLink`s as well? My understanding is Yes based on the conversation had in [the design review doc](https://docs.google.com/document/d/1kqpW1jIJF-h36zrrVPB3AFEf93FYz4wo5rHc0pyxGJ4/edit), relevant to the `Install` element that's shown in designs.

Reply:
So the answer is *yes*, but I'm not adding them in right now, if that's OK!

I think that my plan will be to allow a `onClick` or `url` but not both to be set.  From there we can infer what button type we render.